package.json

```diff
@@ -81,7 +82,8 @@
     "web_modules/"
   ],
   "scripts": {
-    "pretest": "npm run lint && npm run beautify-lint",
+    "lint-file-standard": "eclint check lib/**/*.js hot/*.js bin/*.js benchmark/*.js test/*.js",
+    "pretest": "npm run lint && npm run beautify-lint && npm run lint-file-standard",
```

Comment:
could you add it to `postcover` too? Or split it into a separate script for less code repeatiation.